config.js

```diff
@@ -32,7 +32,7 @@ const defaultConfig = {
   // do not end lines of generated code with ';' if true
   withoutSemi: false,
   // class to use for repeated properties
-  repeatedClass: 'qx.data.Array',
+  repeatedClass: 'Array',
```

Comment:
Is there a reason for this change? Without using qx.data.Array you have no options to use bindings for those properties because native array do not support these. So I guess we should stick to qx.data.Array as default. The code changes related to this can be kept. As since Qx 7.0 qx.data.Array also supports the find method (and that is the only related change I saw).
We might add a minumum requirement for Qx >= 7.0 somewhere in the documentation.